### ISSUE TITLE
Sources not  found or don't support the ddc version: tmux

### ISSUE THREAD

**Issue by jeffwillette** [delphinus/ddc-tmux#6]
With the latest version of ddc I am getting the error:

```
[ddc] Sources not  found or don't support the ddc version: tmux
```

Do you have any idea what would be causing this in the most recent ddc version?

**Comment by delphinus** [comment]
@jeffwillette Probably the problem is fixed by #5. Check it.

**Comment by jeffwillette** [comment]
It is. Thank you
